bridge/matrix/matrix.go

```diff
@@ -372,6 +372,8 @@ func (b *Bmatrix) getAvatarURL(sender string) string {
 		return ""
 	}
 	url := strings.ReplaceAll(mxcURL, "mxc://", b.GetString("Server")+"/_matrix/media/r0/thumbnail/")
```

Comment:
I'd maybe suggest putting the `if mxcUrl == "" { return mxcUrl }` here, to prevent an unnecessary call to `strings.ReplaceAll`